src/Features/CSharp/Portable/UseNamedArguments/CSharpUseNamedArgumentsCodeRefactoringProvider.cs

```diff
@@ -69,7 +69,7 @@ protected override SeparatedSyntaxList<ArgumentSyntax> GetArguments(BaseArgument
                 => argumentList.WithArguments(SyntaxFactory.SeparatedList(namedArguments, separators));
 
             protected override ArgumentSyntax WithName(ArgumentSyntax argument, string name)
-                => argument.WithNameColon(SyntaxFactory.NameColon(name.ToIdentifierName()));
+                => argument.WithoutTrivia().WithNameColon(SyntaxFactory.NameColon(name.ToIdentifierName()));
```

Comment:
so this feels like it's losing the arg trivia taht actually exists.  How does this work out in practice.  like if there is trailing trivia on the argument, won't this lose that?

why aren't we just moving the leading trivia from teh argument to the new `name-colon`?

Reply:
@CyrusNajmabadi We shouldn't be losing trivia. The trivia is indeed added to the new node, but by the caller:

https://github.com/dotnet/roslyn/blob/4cc1b4e445f201eb77e452a20b86df42747b49bf/src/Features/Core/Portable/UseNamedArguments/AbstractUseNamedArgumentsCodeRefactoringProvider.cs#L159

Reply:
I can move the `WithoutTrivia` call to the caller.

Reply:
Done in 1697b22.